.travis.yml

```diff
@@ -23,8 +23,9 @@ env:
   - REQUIREMENTS=devel
 
 python:
-  - "2.7"
-  - "3.5"
+  - "3.6"
+  - "3.7"
+  - "3.8"
```

Comment:
This should also be included in the allowed failures since it's probably going to fail.

Reply:
@Glignos all of them or just `3.8`?

Reply:
@topless just 3.8